packages/bindings/src/serialport_linux.cpp

```diff
@@ -40,29 +40,31 @@ int linuxGetSystemBaudRate(const int fd, int* const outbaud) {
 int linuxSetLowLatencyMode(const int fd, const bool enable) {
   struct serial_struct ss;
 
-  if (ioctl(fd, TIOCGSERIAL, &ss)) {
+  if (ioctl(fd, TIOCGSERIAL, &ss) == -1) {
     return -1;
   }
 
-  if (ss.flags & ASYNC_LOW_LATENCY != enable) {
-    
-    if (enable) {
-      ss.flags |= ASYNC_LOW_LATENCY;
-    } else {
-      ss.flags &= ~ASYNC_LOW_LATENCY;
-    }
+  if (ss.flags & ASYNC_LOW_LATENCY == enable) {
```

Comment:
Not entirely certain why... but the compiler doesn't seem to like this recent change.  At first I thought this was a rogue warning, but it is actually misreading the code if we dont add parentheses around the items per below:
`if ((ss.flags & ASYNC_LOW_LATENCY) == enable) {`

the warnings are:
```
./src/serialport_linux.cpp: In function ‘int linuxSetLowLatencyMode(int, bool)’:
../src/serialport_linux.cpp:47:36: warning: comparison of constant ‘8192’ with boolean expression is always false [-Wbool-compare]
   47 |   if (ss.flags & ASYNC_LOW_LATENCY == enable) {
      |                                    ^
../src/serialport_linux.cpp:47:36: warning: suggest parentheses around comparison in operand of ‘&’ [-Wparentheses]

```
Without adding the parentheses the compiled code does try to set the low latency flag every time, irrespective of the bool enable value that we pass

I guess my c is a little rustier that I thought